### ISSUE TITLE
Problems on linux

### ISSUE THREAD

**Issue by fullmooninu** [magarena/magarena#111]
Min max never seems to play. Monty carlos crashes with the following on strace
```
Score = 844

Exception from controller.runGame: drain failed
java.lang.IllegalStateException: drain failed
        at org.classpath.icedtea.pulseaudio.EventLoop.native_iterate(Native Method)
        at org.classpath.icedtea.pulseaudio.EventLoop.run(EventLoop.java:133)
        at java.lang.Thread.run(Thread.java:745)


 <unfinished ...>
+++ exited with 1 +++
```

So it seems the same as an error I once reported. Disabling sounds seems to fix it.

**Comment by lodici** [comment]
Thanks, I have created a new issue #112 to turn off sound automatically at startup if there is a problem.

**Comment by lodici** [comment]
Addressed in #112.
